common/src/main/java/dev/aurelium/auraskills/common/source/parser/BlockSourceParser.java

```diff
@@ -23,6 +23,7 @@ public BlockSourceParser(AuraSkillsPlugin plugin) {
     @Override
     public BlockSource parse(ConfigurationNode source, ConfigurateSourceContext context) throws SerializationException {
         String[] blocks = context.requiredPluralizedArray("block", source, String.class);
+        int maxBlocks = source.node("maxBlocks").getInt(100);
```

Comment:
For consistency, can you change the name of the option to snake case (max_blocks) and update the sources/foraging.yml file accordingly? Java variables remain camelCase.